<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSetProonnxClass</class>
 <widget class="QDialog" name="DlgSetProonnxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgSetProonnx</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_language">
       <property name="text">
        <string>语言：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="coBox_language">
       <item>
        <property name="text">
         <string>CHN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USA</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_cameraCount">
       <property name="text">
        <string>相机数目:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sBox_cameraCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbtn_accpet">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
